src/livechat.service.js

```diff
@@ -0,0 +1,183 @@
+class LivechatService {
+  /* @ngInject */
+  constructor($http, $location, $q) {
+    this.$http = $http;
+    this.$location = $location;
+    this.$q = $q;
+  }
+
+  getConfiguration() {
+    return this.$http({
+      method: 'GET',
+      url: '/support/chat/configuration',
+    }).then(response => response && response.data)
+      .catch(() => null);
+  }
+
+  getQueue(category, product, sso) {
+    return this.$http({
+      method: 'GET',
+      url: '/support/chat/queue',
+      params: {
+        product,
+        category,
+        sso,
+      },
+    }).then((response) => {
+      const queueConfig = response && response.data;
+
+      if (queueConfig && queueConfig.calendar) {
+        queueConfig.calendar = this.constructor.formatCalendar(queueConfig.calendar);
+      }
+
+      return queueConfig;
+    });
+  }
+
+  static formatCalendar(calendar) {
+    const result = [];
+    [
+      'monday', 'tuesday', 'wednesday',
+      'thursday', 'friday', 'saturday', 'sunday',
+    ].forEach((day) => {
+      if (calendar[day]) {
+        result.push({
+          name: day,
+          slots: calendar[day].map(slot => this.formatCalendarSlot(calendar.timezone, slot)),
+        });
+      }
+    });
+
+    return result;
+  }
+
+  static formatCalendarSlot(timezone, slot) {
+    return {
+      startTime: moment(`${slot.startTime}${timezone}`, 'HH:mm:ssZ').format('LT'),
+      endTime: moment(`${slot.endTime}${timezone}`, 'HH:mm:ssZ').format('LT'),
+    };
+  }
+
+  getAuthentication() {
+    if (!this.$location.host().includes('ovh.com')) {
+      return this.$q.resolve(null);
+    }
+
+    return this.$http({
+      method: 'GET',
+      url: 'https://www.ovh.com/auth/sso/saml/stub.cgi',
```

Comment:
We should probably handle the urls the way we do in other managers (EU/CA/US) event though it's not needed yet, it'll be easier to evolve